src/App.js

```diff
@@ -1,25 +1,43 @@
-import logo from './logo.svg';
-import './App.css';
+import React, {useState} from 'react';
+import {Switch, Route, Redirect} from 'react-router-dom'; 
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Styles/App.css';
+import Home from './components/Home'; 
+import List from './components/List'; 
+
+//add data 
+import data from './data';
+
+const App = () => {
+  //state
+  const [contacts, setContacts] = useState(data()); 
+
+  const addContact = (contact) => {
+    const newContacts = [...contacts];
```

Comment:
The `newContacts` are not new contacts, right? I don't think you need this line. Just spread `...contacts` on line 17